ohhell.py

```diff
@@ -0,0 +1,93 @@
+from collections import defaultdict
+from dataclasses import dataclass
+from typing import List
+
+
+
+DECK_SIZE = 54
+with open("VERSION") as file:
+    GAME_VERSION = file.read()
+SCORE_MULTIPLIER = 10
+
+
+
+@dataclass
+class Player:
+    name: str
+    score: int
+    bids: int
+    wins: int
+
+
+
+def enter_players(num_players: int):
+    players = []
+    for i in range(num_players):
+        players.append(
+            Player(str(input("Enter the player's name: ")), 0, 0, 0)
```

Comment:
Instead of writing the 0s, we could do this?
```python
@dataclass
class Person:
  name: str
  score: int = 0
  wins: int = 0 
  bids: int = 0
```